api/HTMLTrackElement.json

```diff
@@ -75,17 +78,31 @@
               "version_added": false
             },
             "opera": {
-              "version_added": "15"
+              "version_added": "12"
             },
             "opera_android": {
-              "version_added": "14"
-            },
-            "safari": {
-              "version_added": "7"
-            },
-            "safari_ios": {
-              "version_added": "7"
+              "version_added": "12"
             },
+            "safari": [
+              {
+                "version_added": "10"
+              },
+              {
+                "version_added": "6",
+                "partial_implementation": true,
+                "notes": "The <code>oncuechange</code> event handler property is not supported."
```

Comment:
We'll need a note like this for Chrome and friends too. Using http://software.hixie.ch/utilities/js/live-dom-viewer/?saved=10032 I found that it was added in Chrome 32.